plinth/templates/base.html

```diff
@@ -222,5 +222,6 @@
 
   {% block app_js %}<!-- placeholder for app-specific js files -->{% endblock %}
   {% block page_js %}<!-- placeholder for page-specific js files -->{% endblock %}
+  <a href="/static/jslicense.html" data-jslicense="1">JavaScript license information</a>
```

Comment:
Ah we should use the static tag, something like this:
```
href="{% static 'jslicense.html' %}"
```
Then Django should pick the correct folder where static files are being provided.